spec/integ/crypto/crypto.spec.ts

```diff
@@ -2329,5 +2374,30 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
                 expect(createSecretStorageKey).toHaveBeenCalledTimes(2);
             },
         );
+
+        newBackendOnly("should upload cross signing master key", async () => {
+            mockSetupCrossSigningRequests();
+
+            await aliceClient.getCrypto()?.bootstrapCrossSigning({});
```

Comment:
```suggestion
            // Before setting up secret-storage, bootstrap cross-signing, so that the client has cross-signing keys.
            await aliceClient.getCrypto()?.bootstrapCrossSigning({});
```